adal/devApps/XFormsApp/App.cs

```diff
@@ -52,7 +52,7 @@ protected override void OnStart()
             // Handle when your app starts
             AppCenter.Start("android=3ea693de-dcb4-4b65-a1b5-04ad2f71ada0;" +
                   "uwp={Your UWP App secret here};" +
-                  "ios={Your iOS App secret here}",
+                  "ios={4be201cc-2027-4045-b535-d57de5ff36c7}",
```

Comment:
Should this be checked in?

Reply:
This is simply the appID of our application in appcenter and it is used for identification purposes only. All appcenter credentials are stored securely elsewhere.

Reply:
nit: should the app id be in KeyVault? Allowing it to be changed outside of the test environment?

---
In reply to: [235211318](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1418#discussion_r235211318) [](ancestors = 235211318)